app/extensions/ipfs/js/main.js

```diff
@@ -0,0 +1,54 @@
+/* global Ipfs */
+
+chrome.protocol.registerStringProtocol('ipfs', handler)
+
+const node = new Ipfs({
+  config: {
+    Addresses: {
+      Swarm: []
+    }
+  }
+})
+
+// TODO: bring back once brave supports registerURIHandlers
```

Comment:
@diasdavid do you have any background info on why `registerURIHandlers` is needed?

Reply:
because an IPFS hash is, in reality, a URI and not a URL, we should be able to do `ipfs:QmHash` and not `ipfs://QmHash`.

This is still an ongoing proposal, read more at:
- https://github.com/ipfs/specs/pull/152
- https://github.com/ipfs/specs/pull/167/files